templates/confidenceSlider.jsx

```diff
@@ -1,25 +1,82 @@
+import React, { useEffect, useRef, useState } from 'react';
 import Adapt from 'core/js/adapt';
-import React from 'react';
 import { classes, templates } from 'core/js/reactHelpers';
 
-export default function ConfidenceSlider (props) {
+export default function Slider (props) {
```

Comment:
This should probably be kept as ConfidenceSlider

Reply:
Quite right; the perils of copy and paste!